src/content/docs/extensions/iceberg.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: "iceberg extension"
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+## Usage
+
+The `iceberg` extension adds support for `iceberg` objects, currently including scanning from [Apache Iceberg format](https://iceberg.apache.org/).
+
+The iceberg functionality is not available by default, so you would first need to install the iceberg
+extension by running the following commands:
+
+```sql
+INSTALL iceberg;
+LOAD EXTENSION iceberg;
+```
+
+### Example dataset
+
+To test the examples, download the [iceberg_data.zip](TODO: upload iceberg sample data for download) file and unzip it.
+
+## Usage
+
+### Common Parameters
+
+```
+┌───────────────────────────────┬───────────┬──────────────┬──────────────────────────────────────────────────────────────────────────────┐
+│ Parameter                     │ Type      │ Default      │ Description                                                                  │
+├───────────────────────────────┼───────────┼──────────────┼──────────────────────────────────────────────────────────────────────────────┤
+│ allow_moved_paths             │ BOOLEAN   │ false        │ Allows scanning Iceberg tables that are moved                                │
+│ metadata_compression_codec    │ STRING    │ ''           │ Treats metadata files as when set to 'gzip'                                  │
+│ version                       │ STRING    │ '?'          │ Provides an explicit version string, hint file or guessing                   │
+│ version_name_format           │ STRING    │ 'v%s%s.metadata.json,%s%s.metadata.json' │ Controls how versions are converted to metadata file names │
```

Comment:
I think the Options table is too brief to explain the idea behind these options, you should have a separate paragraph on explaining each option more clearly. 
Each option should come with an example query to demo how they are set/used.

When I go over this as someone not an expert on iceberg, I got confused:
1. What does moved paths mean? and how the behaviour would be when `allow_moved_paths` is set to `true` or `false`, respectively?
2. What compression codec do we support for `metadata_compression_codec`? Default value `''` means uncompressed?
3. What are the rules for version name format?

Reply:
restructured this